<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartWindow</class>
 <widget class="QDialog" name="ChartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1282</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lie Detector</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>1131</width>
     <height>651</height>
    </rect>
   </property>
   <property name="title">
    <string>Live charts: Top - GSR, Middle - Pneumograph, Bottom - Pulse</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QCustomPlot" name="widgetGSR" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="QCustomPlot" name="widgetPneumograph" native="true"/>
    </item>
    <item row="2" column="0">
     <widget class="QCustomPlot" name="widgetECG" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>10</y>
     <width>117</width>
     <height>651</height>
    </rect>
   </property>
   <property name="title">
    <string>Menu</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Set mark</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButtonPause">
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButtonResume">
      <property name="text">
       <string>Resume</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButtonClearChart">
      <property name="text">
       <string>Clean</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="SavetoFile">
      <property name="text">
       <string>Save to File</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
